cloudigrade/config/settings/base.py

```diff
@@ -40,6 +40,64 @@
     'HOUNDIGRADE_AWS_VOLUME_BATCH_SIZE',
     default=32
 )
+HOUNDIGRADE_ECS_CLUSTER_NAME = env(
+    'HOUNDIGRADE_ECS_CLUSTER_NAME',
+    default='inspectigrade-test-bws-us-east-1b'
+)
+HOUNDIGRADE_ECS_FAMILY_NAME = env(
+    'HOUNDIGRADE_ECS_FAMILY_NAME',
+    default='Houndigrade'
+)
+HOUNDIGRADE_ECS_IMAGE_NAME = env(
+    'HOUNDIGRADE_ECS_IMAGE_NAME',
+    default='cloudigrade/houndigrade'
+)
+HOUNDIGRADE_ECS_IMAGE_TAG = env(
+    'HOUNDIGRADE_ECS_IMAGE_TAG',
+    default='latest'
+)
+HOUNDIGRADE_MOUNT_POINTS = env(
+    'HOUNDIGRADE_MOUNT_POINTS',
+    default=[
+        '/dev/xvdb',
```

Comment:
That looks like the right number of entries, which is a perfect match to the default `HOUNDIGRADE_AWS_VOLUME_BATCH_SIZE`, but I wonder if we should generate this list based on that value instead of 32 lines here. I don't feel strongly about this; just throwing out an idea.

Reply:
List has now been replaced with `util.generate_device_name()`. Give it an index, and optionally a prefix, and it spits out a device name for you.

Reply:
The spec for the device names on AWS, https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/device_naming.html#available-ec2-device-names . We're using HVM.